test/PathSpec.hs

```diff
@@ -38,6 +38,11 @@ spec = do
         it "Empty path" $ rt "/"
         it "Top level path" $ rt "/fdoo"
         it "Nested path" $ rt "/boo_1/chew_2"
+    describe "isChildOf" $ do
+        it "Child -> True" $ isChildOf [pathq|/a/b/c/d|] [pathq|/a|] `shouldBe` True
```

Comment:
Might be more canonical to write `[pathq|/a/b/c/d|] ``shouldSatisfy`` isChildOf [pathq|/a|]` and likewise with `shouldNotSatisfy`